enterprise/conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -577,14 +577,95 @@ msgid ""
 "course, a program, or something else.</li></ul></ul>"
 msgstr ""
 
-#: enterprise/models.py:1007
+#: enterprise/models.py:1035
 #, python-brace-format
 msgid ""
 "Enter a string that can be used to generate a dynamic subject line for "
 "notification emails. The placeholder {course_name} will be replaced with the "
 "name of the course or program that was enrolled in."
 msgstr ""
 
+#: enterprise/models.py:1128
+msgid "Active"
+msgstr ""
+
+#: enterprise/models.py:1134
+msgid "Delivery Method"
+msgstr ""
+
+#: enterprise/models.py:1135
+msgid "The method in which the data should be sent."
+msgstr ""
+
+#: enterprise/models.py:1137
+msgid "Email"
+msgstr ""
+
+#: enterprise/models.py:1143
+msgid "Frequency"
+msgstr ""
+
+#: enterprise/models.py:1144
+msgid ""
+"The frequency interval (daily, weekly, or monthly) that the report should be "
+"sent."
+msgstr ""
+
+#: enterprise/models.py:1149
+msgid "Day of Month"
+msgstr ""
+
+#: enterprise/models.py:1150
+msgid ""
+"The day of the month to send the report. This field is required and only "
+"valid when the frequency is monthly."
+msgstr ""
+
+#: enterprise/models.py:1158
+msgid "Day of Week"
+msgstr ""
+
+#: enterprise/models.py:1159
+msgid ""
+"The day of the week to send the report. This field is required and only "
+"valid when the frequency is weekly."
+msgstr ""
+
+#: enterprise/models.py:1163
+msgid "Hour of Day"
+msgstr ""
+
+#: enterprise/models.py:1164
+msgid ""
+"The hour of the day to send the report, in Eastern Standard Time (EST). This "
+"is required for all frequency settings."
+msgstr ""
+
+#: enterprise/models.py:1173
+msgid "Password Encryption Initialization Vector"
+msgstr ""
+
+#: enterprise/models.py:1174
+msgid ""
+"This is the initialization vector used to encrypt the password using AESencr"
+msgstr ""
+
+#: enterprise/models.py:1181
+msgid "Password"
+msgstr ""
+
+#: enterprise/models.py:1222
+msgid "Day of week must be set if the frequency is weekly."
+msgstr ""
+
+#: enterprise/models.py:1226
+msgid "Day of month must be set if the frequency is monthly."
+msgstr ""
+
+#: enterprise/models.py:1229
+msgid "Frequency must be set to either daily, weekly, or monthly."
+msgstr ""
+
```

Comment:
@UmanShahzad This PR is code removal only (aside from the migration that removes model data), so why add any contents here? :)

Reply:
I didn't -- `make extract_translations` did. I think someone just forgot to run it previously.